tests/test_cli.py

```diff
@@ -32,28 +18,35 @@ def run_cli_cmd(command: str, working_dir: typing.Union[str, Path]):
 )
 def test_nebari_init(tmp_path, namespace, auth_provider, ci_provider, ssl_cert_email):
     """Test `nebari init` CLI command."""
-    command = f"nebari init local --project {PROJECT_NAME} --domain {DOMAIN_NAME} --disable-prompt"
+    command = [
```

Comment:
Minor refactoring. Instead of building a string here and splitting it again afterwards, let's just use a list from the start.